src/mmrelay/matrix/media.py

```
@@ -0,0 +1,144 @@
+
+    try:
+        response, _ = await client.upload(
+            io.BytesIO(image_data),
+            content_type=content_type,
+            filename=filename,
+            filesize=len(image_data),
+        )
+    except facade.NIO_COMM_EXCEPTIONS as e:
+        facade.logger.exception("Image upload failed due to a network error")
+        upload_error = SimpleNamespace(message=str(e), status_code=None)
+        return upload_error
```

Ruff (0.15.9)

[warning] Unnecessary assignment to `upload_error` before `return` statement

Remove unnecessary assignment

(RET504)